src/algebra/category/Algebra/limits.lean

```diff
@@ -53,7 +53,9 @@ end
 instance limit_algebra (F : J ⥤ Algebra R) :
   algebra R (types.limit_cone (F ⋙ forget (Algebra.{v} R))).X :=
 begin
-  change algebra R (sections_subalgebra F),
+  have : algebra R (types.limit_cone (F ⋙ forget (Algebra.{v} R))).X
+    = algebra R (sections_subalgebra F), by refl,
+  rw this,
```

Comment:
Wow, it's sad that this speeds things up.

Reply:
Yes, a lot of things in this PR are really weird. Here, it even changes the algebra structure that is inferred (see the next change in the PR, adding a convert).